doc/changes/changes_0.3.1.md

```diff
@@ -0,0 +1,11 @@
+# Artifact Reference Checker Maven Plugin 0.3.1, released 2020-09-08
+
+Code name: Changes default phase to package
+
+## Summary
```

Comment:
Don't add a summary if you don't have any additional information to say here, it's redundant.